ui/shared/nft/NftImageFullscreen.tsx

```diff
@@ -11,12 +11,21 @@ interface Props {
   onClose: () => void;
 }
 
-const NftImageWithFullscreen = ({ src, isOpen, onClose }: Props) => {
+const NftImageFullscreen = ({ src, isOpen, onClose }: Props) => {
+  const imgRef = React.useRef<HTMLImageElement>(null);
+
+  const checkWidth = React.useCallback(() => {
+    if (imgRef.current?.getBoundingClientRect().width === 0) {
+      imgRef.current.style.width = '90vw';
+      imgRef.current.style.height = '90vh';
```

Comment:
Shouldn't we set these properties through React component props instead of directly manipulating the DOM tree?